### DISCUSSION TITLE
Not available on Dell U3415W

### DISCUSSION THREAD

**Discussion by tedshim** [MonitorControl/MonitorControl#1271] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

 Not available on Dell U3415W

### Expected behavior

available on Dell U3415W

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.6 macOS Monterey
- Mac model: MacBook Pro (16-inch, 2019)
- MonitorControl version: 4.1.0
- Monitor(s): Dell U3415W
- Monitor Cable(s)/Connection(s): Mac book Usb 3.0 -> Displayport 2.1 -> Dell U3415W
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
if you are using a standard USB 3.0 to DisplayPort adapter then it is possible a DisplayLink adapter which does not support DDC communications on the Mac. Your only option is software dimming. Alternatively you can try [BetterDisplay](https://github.com/waydabber/BetterDisplay#readme) and force Color table manipulation in Settings/Displays as newer DisplayLink drivers do support this more advanced form of dimming.
